front/src/applications/stdcm/utils.ts

```diff
@@ -0,0 +1,36 @@
+function generateRandomString(length: number): string {
+  return [...Array(length)].map(() => Math.floor(Math.random() * 10)).join('');
```

Comment:
You only need an array to use map and Array(length) is already one
```suggestion
  return Array(length).map(() => Math.floor(Math.random() * 10)).join('');
```

Reply:
done